### PR TITLE
DRG: Improve folder structure,unify UI and added dwarf promotion rank selection

### PR DESCRIPTION
**Pull request by michyaraque**
The reason for changing the folder architecture is that when navigating between folders having only index.tsx files makes it tedious.

### CODE REVIEW

**Review by MrMarble — CHANGES_REQUESTED** [review]
The screenshot is made with an older revision of the code.
  - Resources icon
  - Enabled tabs

File: src/components/UploadFile/UploadFile.tsx [hunk 1]
```diff
@@ -45,8 +45,10 @@ export const SaveUpload = () => {
       className="rounded m-6 border-dashed border-drg-primary-700 border-2 p-6 text-center cursor-pointer select-none hover:bg-drg-primary-400 hover:text-slate-900 transition-all duration-200"
     >
       <input {...getInputProps()} />
-      <p>Drag 'n' drop your save here, or click to select files</p>
+      <p>{`Drag 'n' drop your save here, or click to select files`}</p>
```
Comment (MrMarble) [inline src/components/UploadFile/UploadFile.tsx:48]: Why?

Comment (michyaraque) [reply src/components/UploadFile/UploadFile.tsx:48]: The linter has asked for it **"react/no-unescaped-entities"**

Comment (MrMarble) [reply src/components/UploadFile/UploadFile.tsx:48]: And where is the entity? Eslint doesn't fires for me

**Pull request merged**